# Source and destination IDs
string goalType
float64 source_id
float64 target_id

---
string result